com.unity.testing.urp/package.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "com.unity.testing.urp",
+	"displayName":"URP Graphic Tests Library",
+	"version": "1.0.0",
+	"unity": "2019.3",
+	"unityRelease": "0b4",
+	"description": "Common assets library for URP test projects.",
+	"keywords": ["qa", "test", "testing", "tests", "graphics", "HDRP"],
+	"dependencies": {
+	"com.unity.testframework.graphics": "7.8.7-preview",
+	"com.unity.render-pipelines.universal": "file:./com.unity.render-pipelines.universal"
```

Comment:
Is a dot missing in the path? I'd expect the package to live in the parent folder, such as `../com.unity.render-pipelines.universal`

Reply:
Edit: Looking at the testing.hdrp package.json I see only one dot too, so discard what I said

Reply:
The single `.` means current folder and `..` parent folder.